v22.1/migrate-from-postgres.md

```diff
@@ -117,9 +115,9 @@ This example assumes you [dumped the entire database](#dump-the-entire-database)
 
 ### Import a table from a table dump
 
-The examples below assume you [dumped one table](#dump-one-table-at-a-time).
+The following examples assume you [dumped one table](#dump-one-table-at-a-time).
```

Comment:
doesn't necessarily need to be the case. Doing `IMPORT TABLE xxx` means that itll just pickout the statements for that table even if the dump file has many more tables.